### PR TITLE
LICENSE dialog added

### PR DESCRIPTION
**Pull request by stefanodellosa-personal**
Complete WADAS menu with corresponding dialogs.

### CODE REVIEW

File: wadas/ui/about_dialog.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
+"""Module containing logic to show About WADAS info in dedicated dialog."""
+
+import os
+
+from PySide6.QtWidgets import QDialog, QVBoxLayout, QTextBrowser, QPushButton
+from PySide6.QtGui import QIcon
```
Comment (cdimauro) [inline wadas/ui/about_dialog.py:6]: Sort imports (it looks that isort, on the pre-commit, doesn't work).

File: wadas/ui/about_dialog.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+        )
+        about_text.setOpenExternalLinks(True)  # Allow opening links in a browser
+        layout.addWidget(about_text)
+
+        # Add a close button
+        close_button = QPushButton("Close", self)
+        close_button.clicked.connect(self.close)
+        layout.addWidget(close_button)
+
+        # Set the layout
```
Comment (cdimauro) [inline wadas/ui/about_dialog.py:54]: Redundant

File: wadas/ui/license_dialog.py [hunk 3]
```diff
@@ -0,0 +1,38 @@
+"""Module containing logic to draw license dialog."""
+
+import os
+
+from PySide6.QtCore import Qt
+from PySide6.QtWidgets import QDialog, QVBoxLayout, QPushButton, QTextEdit
+from PySide6.QtGui import QIcon
```
Comment (cdimauro) [inline wadas/ui/license_dialog.py:7]: Sort imports.

File: wadas/ui/license_dialog.py [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+        self.text_edit = QTextEdit(self)
+        self.text_edit.setReadOnly(True)
+        self.text_edit.setVerticalScrollBarPolicy(Qt.ScrollBarAlwaysOn)
+        layout.addWidget(self.text_edit)
+
+        # Load license file
+        license_dir = os.path.abspath(os.path.join(module_dir_path, "..", ".."))
+        license_path = os.path.join(license_dir, "LICENSE")
+        with open(license_path, 'r') as file:
```
Comment (cdimauro) [inline wadas/ui/license_dialog.py:30]: 'r' is redudant (it's the default).

Comment (cdimauro) [inline wadas/ui/license_dialog.py:30]: Rename file to file_ (file is a builtin).

**Review by hexfati — APPROVED** [review]

**Pull request merged**